### PR TITLE
issue #5972 Hover action in the icons and logo of NavBar solved

### LINKED ISSUE #5972: Hover action in the icons and logo of NavBar
### Description

The color of the Github icon and the the dark mode toggle icon is white when compared to the other links. And there is also no action performed while hovering over the icon buttons apart from the mouse pointer functionality. This looks quite odd. And the same applies also to the logo at the extreme left of the Nav Bar

### Screenshots

image

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by anupamthe99**
## Fixes Issue
Hover action in the icons and logo of NavBar #5972

fixes #5972 

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] All new and existing tests passed.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5974"></a>

### CODE REVIEW

**Review by TRohit20 — COMMENTED** [review]
HI @anupamthe99 , Thank you for the contribution, Just a small suggestion, When you make code contributions like this, it is always better to add screenshots and make it more clear for the reviewers and maintainers. :)

File: components/navbar/Navbar.js [hunk 1]
```diff
@@ -170,7 +170,8 @@ export default function Navbar() {
                     rel="noreferrer"
                     className="text-white"
                   >
-                    <FaGithub aria-label="GitHub" />
+                    <FaGithub className="text-gray-400 hover:text-white" aria-label="GitHub" />
+
```
Comment (eddiejaoude) [inline components/navbar/Navbar.js:174]: I don't think the extra blank link is required

Comment (eddiejaoude) [reply components/navbar/Navbar.js:174]:
```suggestion
```

File: components/navbar/Navbar.js [hunk 2]
```diff
@@ -38,9 +38,9 @@ export default function Navbar() {
     if (currentTheme === "dark") {
       return (
         <button className="p-2 text-white" onClick={() => setTheme("light")}>
-          <FaSun />
+          <FaSun className="text-gray-400 hover:text-white"/>
```
Comment (eddiejaoude) [inline components/navbar/Navbar.js:41]: We are trying to not use `...gray-400`, but our predefined colors in the tailwind config

Comment (eddiejaoude) [reply components/navbar/Navbar.js:41]: I noticed, in dark mode we have have no borders around the buttons, so I will fix that and update the `-400`

**Pull request merged**